provisioning/service/tests/Config/IndividualEnrollmentTests.cs

```diff
@@ -259,6 +264,7 @@ public void IndividualEnrollmentConstructorJSONSucceed()
             Assert.AreEqual(SampleCreateDateTimeUTC, individualEnrollment.CreatedDateTimeUtc);
             Assert.AreEqual(SampleLastUpdatedDateTimeUTC, individualEnrollment.LastUpdatedDateTimeUtc);
             Assert.AreEqual(SampleEtag, individualEnrollment.ETag);
+            Assert.AreEqual(SampleEdgeCapability.IotEdge, individualEnrollment.Capabilities.IotEdge);
```

Comment:
We should enhance the existing test (3rd test) to assert something if the Capability was not set (should be either `false` or null).